projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -105,39 +98,28 @@ export class NavigationListComponent implements OnChanges {
     if (changes.navItems) {
       this.navItems = this.navListComponentService.resolveFeaturesAndUpdateVisibilityForNavItems(this.navItems);
 
-      // If the FFs are enabled, decorate the nav items with the corresponding time ranges.
+      // Decorate the nav items with the corresponding time ranges, depending on the FF state.
       // The time ranges in nav items are streams that get the most recent value from page time range preference service
       this.navItems$ = this.featureStateResolver
         .getCombinedFeatureState([ApplicationFeature.PageTimeRange, ApplicationFeature.NavigationRedesign])
```

Comment:
It's still unclear to me why these two things are combined. the time range being set per page seems independent of the navigation.